installer/functions/task-router/task-router-helpers.private.ts

```diff
@@ -0,0 +1,91 @@
+import { Context, TwilioClient } from '@twilio-labs/serverless-runtime-types/types';
+import { WorkspaceInstance } from 'twilio/lib/rest/taskrouter/v1/workspace';
+import { TaskQueueInstance } from 'twilio/lib/rest/taskrouter/v1/workspace/taskQueue';
+import { WorkerInstance } from 'twilio/lib/rest/taskrouter/v1/workspace/worker';
+import { WorkflowInstance } from 'twilio/lib/rest/taskrouter/v1/workspace/workflow';
+
+export const WORKSPACE_FRIENDLY_NAME = "Flex Task Assignment";
+export const SCHEDULERS = "Schedulers";
+export const EDUCATORS = "Educators";
+export const SCHEDULING = "Scheduling";
+export const EDUCATION = "Education";
+export const INTAKE_BY_SCHEDULERS = "Intake by Schedulers";
+export const TRANSFER_TO_NURSE_EDUCATOR = "Transfer to Nurse Educator";
+
+export interface QueueToQueueSid {
+  schedulerSid: string;
+  educatorSid: string;
+}
+
+// Function which creates Flex Workspace
+export function createWorkspace(client: TwilioClient, friendlyName: string): Promise<WorkspaceInstance> {
+  const workspace = client.taskrouter.workspaces
+        .create({
+          template: 'FIFO',
+          friendlyName
+        })
+        .then(workspace => workspace);
+  return workspace;
+}
+
+// Creates task queue given a workspace SID
+export function createTaskQueue(client: TwilioClient, friendlyName: string, workspaceSid: string, skill?: string): Promise<TaskQueueInstance> {
+  const taskQueue = client.taskrouter.workspaces(workspaceSid)
+    .taskQueues
+    .create({
+      friendlyName,
+      targetWorkers: skill ? `routing.skills HAS "${skill}"` : "1==1"
+    })
+    .then(task_queue => task_queue);
+  return taskQueue;
```

Comment:
same comment as above and for essentially the rest of the functions in this file